scripts/build_wheel.py

```
@@ -545,6 +570,129 @@ def build_kv_cache_manager_v2(project_dir,
+    if tar_bin is None:
+        return False
+    dst.mkdir(parents=True, exist_ok=True)
+    # posix (pax) format keeps sub-second mtimes; the gnu default truncates
+    # to whole seconds, which would defeat sync_tree's mtime comparison.
+    #
+    # Run under bash with pipefail so a failing producer tar (e.g. an
+    # unreadable source file) is reported instead of being masked by the
+    # consumer tar's exit status; plain shell=True uses /bin/sh (dash), where
+    # pipefail is not guaranteed. On failure the caller falls back to copytree.
+    bash = shutil.which("bash") or "/bin/bash"
+    result = run(
```

Ruff (0.16.1)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)